src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperExecutionContext.cs

```diff
@@ -134,6 +134,26 @@ public void Add([NotNull] ITagHelper tagHelper)
         }
 
         /// <summary>
+        /// Tracks the minimized HTML attribute in <see cref="AllAttributes"/> and <see cref="HTMLAttributes"/>.
+        /// </summary>
+        /// <param name="name">The minimized HTML attribute name.</param>
+        public void AddHtmlAttribute([NotNull] string name)
```

Comment:
This warrants a different name because it's doing something different.  In particular, the effect is not the same as a call to `AddHtmlAttribute("attribute-name", value: null)` (the basic expectation of overloads).  So please rename it to `AddMinimizedAttribute()` or `AddMinimizedHtmlAttribute()`.